exercises/e-data02-derive-master.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        &gt;&gt; gr.tf_mutate(
</pre><p>This should be a summarize, since you're using summary functions inside the call.</p><p> </p><p>A rule of thumb is that, if you're using summary functions, you should be using summarize. One exception is if you're trying to normalize every value by a summary, e.g.</p><p> </p><p><code>carat_norm = DF.carat / gr.mean(DF.carat)</code> - which we'd put in a mutate </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdelrosario/evc-course/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>